unhandled exception java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'

### Steps to reproduce

1. Install webtoons extension
2. Configure extension to use "Show author's notes"
3. [Try downloading chapters 14+](https://www.webtoons.com/en/fantasy/archmage-curriculum/list?title_no=9884)
4.  Download stops at 99%
<details>

<summary>unhandled exception java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'</summary>

```java
20:44:10.178 [OkHttp Dispatcher] ERROR suwayomi.tachidesk.server.ServerSetup -- unhandled exception
java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at keiyoushi.a.b.a.a(Unknown Source)
	at keiyoushi.a.b.a.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.extension.all.webtoons.z.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.CloudflareInterceptor.intercept(CloudflareInterceptor.kt:48)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(UserAgentInterceptor.kt:21)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(UncaughtExceptionInterceptor.kt:18)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:232)
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:580)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
20:44:10.178 [DefaultDispatcher-worker-7] WARN suwayomi.tachidesk.manga.impl.download.Downloader source(2522335540328470744) - downloadChapter(4153 - 39776 | state= Queued, tries= 0, progress= 0.0)) -- failed due to
java.io.IOException: canceled due to java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at eu.kanade.tachiyomi.source.online.HttpSource.getImage(HttpSource.kt:424)
	at suwayomi.tachidesk.manga.impl.Page$getPageImage$6.invokeSuspend(Page.kt:139)
	at suwayomi.tachidesk.manga.impl.Page$getPageImage$6.invoke(Page.kt)
	at suwayomi.tachidesk.manga.impl.Page$getPageImage$6.invoke(Page.kt)
	at suwayomi.tachidesk.manga.impl.util.storage.ImageResponse.getImageResponse(ImageResponse.kt:69)
	at suwayomi.tachidesk.manga.impl.Page.getPageImage(Page.kt:138)
	at suwayomi.tachidesk.manga.impl.Page.getPageImage$default(Page.kt:58)
	at suwayomi.tachidesk.manga.impl.Page.getPageImageDownload(Page.kt:189)
	at suwayomi.tachidesk.manga.impl.download.fileProvider.ChaptersFilesProvider.downloadImpl(ChaptersFilesProvider.kt:150)
	at suwayomi.tachidesk.manga.impl.download.fileProvider.ChaptersFilesProvider.access$downloadImpl(ChaptersFilesProvider.kt:67)
	at suwayomi.tachidesk.manga.impl.download.fileProvider.ChaptersFilesProvider$downloadImpl$1.invokeSuspend(ChaptersFilesProvider.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:807)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
Caused by: java.io.IOException: canceled due to java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:593)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at keiyoushi.a.b.a.a(Unknown Source)
	at keiyoushi.a.b.a.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.extension.all.webtoons.z.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.CloudflareInterceptor.intercept(CloudflareInterceptor.kt:48)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(UserAgentInterceptor.kt:21)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(UncaughtExceptionInterceptor.kt:18)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:232)
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:580)
	... 3 common frames omitted
20:44:10.178 [DefaultDispatcher-worker-7] DEBUG suwayomi.tachidesk.manga.impl.download.Downloader source(2522335540328470744) - downloadChapter(4153 - 39776 | state= Queued, tries= 1, progress= 0.9925926)) -- start
20:44:10.180 [DefaultDispatcher-worker-7] DEBUG suwayomi.tachidesk.manga.impl.chapter.ChapterForDownload(mangaId= 4153, chapterId= 39776, chapterIndex= 14)::asDownloadReady -- isMarkedAsDownloaded= false, dbPageCount= 135, downloadPageCount= 0
20:44:10.180 [DefaultDispatcher-worker-7] DEBUG suwayomi.tachidesk.manga.impl.chapter.ChapterForDownload(mangaId= 4153, chapterId= 39776, chapterIndex= 14)::asDownloadReady -- reset download status and fetch page list
20:44:10.450 [DefaultDispatcher-worker-7] DEBUG eu.kanade.tachiyomi.network.NetworkHelper -- --> GET https://www.webtoons.com/en/fantasy/archmage-curriculum/ep-14-the-meeting/viewer?title_no=9884&episode_no=14 h2
20:44:11.709 [DefaultDispatcher-worker-7] DEBUG eu.kanade.tachiyomi.network.NetworkHelper -- <-- 200 https://www.webtoons.com/en/fantasy/archmage-curriculum/ep-14-the-meeting/viewer?title_no=9884&episode_no=14 (1258ms, unknown-length body)
20:44:11.759 [OkHttp Dispatcher] ERROR suwayomi.tachidesk.server.ServerSetup -- unhandled exception
java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at keiyoushi.a.b.a.a(Unknown Source)
	at keiyoushi.a.b.a.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.extension.all.webtoons.z.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.CloudflareInterceptor.intercept(CloudflareInterceptor.kt:48)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(UserAgentInterceptor.kt:21)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(UncaughtExceptionInterceptor.kt:18)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:232)
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:580)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
20:44:11.759 [DefaultDispatcher-worker-7] WARN suwayomi.tachidesk.manga.impl.download.Downloader source(2522335540328470744) - downloadChapter(4153 - 39776 | state= Queued, tries= 1, progress= 0.9925926)) -- failed due to
java.io.IOException: canceled due to java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at eu.kanade.tachiyomi.source.online.HttpSource.getImage(HttpSource.kt:424)
	at suwayomi.tachidesk.manga.impl.Page$getPageImage$6.invokeSuspend(Page.kt:139)
	at suwayomi.tachidesk.manga.impl.Page$getPageImage$6.invoke(Page.kt)
	at suwayomi.tachidesk.manga.impl.Page$getPageImage$6.invoke(Page.kt)
	at suwayomi.tachidesk.manga.impl.util.storage.ImageResponse.getImageResponse(ImageResponse.kt:69)
	at suwayomi.tachidesk.manga.impl.Page.getPageImage(Page.kt:138)
	at suwayomi.tachidesk.manga.impl.Page.getPageImage$default(Page.kt:58)
	at suwayomi.tachidesk.manga.impl.Page.getPageImageDownload(Page.kt:189)
	at suwayomi.tachidesk.manga.impl.download.fileProvider.ChaptersFilesProvider.downloadImpl(ChaptersFilesProvider.kt:150)
	at suwayomi.tachidesk.manga.impl.download.fileProvider.ChaptersFilesProvider.access$downloadImpl(ChaptersFilesProvider.kt:67)
	at suwayomi.tachidesk.manga.impl.download.fileProvider.ChaptersFilesProvider$download$1.execute(ChaptersFilesProvider.kt:212)
	at suwayomi.tachidesk.manga.impl.download.fileProvider.ChaptersFilesProvider$download$1.execute(ChaptersFilesProvider.kt:212)
	at suwayomi.tachidesk.manga.impl.ChapterDownloadHelper.download(ChapterDownloadHelper.kt:49)
	at suwayomi.tachidesk.manga.impl.download.Downloader.run(Downloader.kt:155)
	at suwayomi.tachidesk.manga.impl.download.Downloader.access$run(Downloader.kt:40)
	at suwayomi.tachidesk.manga.impl.download.Downloader$run$1.invokeSuspend(Downloader.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:98)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:807)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
Caused by: java.io.IOException: canceled due to java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:593)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at keiyoushi.a.b.a.a(Unknown Source)
	at keiyoushi.a.b.a.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.extension.all.webtoons.z.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.CloudflareInterceptor.intercept(CloudflareInterceptor.kt:48)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(UserAgentInterceptor.kt:21)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(UncaughtExceptionInterceptor.kt:18)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:232)
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:580)
	... 3 common frames omitted
20:44:11.760 [DefaultDispatcher-worker-7] DEBUG suwayomi.tachidesk.manga.impl.download.Downloader source(2522335540328470744) - downloadChapter(4153 - 39776 | state= Queued, tries= 2, progress= 0.9925926)) -- start
20:44:11.761 [DefaultDispatcher-worker-7] DEBUG suwayomi.tachidesk.manga.impl.chapter.ChapterForDownload(mangaId= 4153, chapterId= 39776, chapterIndex= 14)::asDownloadReady -- isMarkedAsDownloaded= false, dbPageCount= 135, downloadPageCount= 0
20:44:11.761 [DefaultDispatcher-worker-7] DEBUG suwayomi.tachidesk.manga.impl.chapter.ChapterForDownload(mangaId= 4153, chapterId= 39776, chapterIndex= 14)::asDownloadReady -- reset download status and fetch page list
20:44:11.762 [DefaultDispatcher-worker-7] DEBUG eu.kanade.tachiyomi.network.NetworkHelper -- --> GET https://www.webtoons.com/en/fantasy/archmage-curriculum/ep-14-the-meeting/viewer?title_no=9884&episode_no=14 h2
20:44:11.782 [DefaultDispatcher-worker-7] DEBUG eu.kanade.tachiyomi.network.NetworkHelper -- <-- 200 https://www.webtoons.com/en/fantasy/archmage-curriculum/ep-14-the-meeting/viewer?title_no=9884&episode_no=14 (19ms, 20421-byte body)
20:44:11.813 [OkHttp Dispatcher] ERROR suwayomi.tachidesk.server.ServerSetup -- unhandled exception
java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at keiyoushi.a.b.a.a(Unknown Source)
	at keiyoushi.a.b.a.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.extension.all.webtoons.z.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.CloudflareInterceptor.intercept(CloudflareInterceptor.kt:48)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(UserAgentInterceptor.kt:21)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(UncaughtExceptionInterceptor.kt:18)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:232)
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:580)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
20:44:11.813 [DefaultDispatcher-worker-7] WARN suwayomi.tachidesk.manga.impl.download.Downloader source(2522335540328470744) - downloadChapter(4153 - 39776 | state= Queued, tries= 2, progress= 0.9925926)) -- failed due to
java.io.IOException: canceled due to java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at eu.kanade.tachiyomi.source.online.HttpSource.getImage(HttpSource.kt:424)
	at suwayomi.tachidesk.manga.impl.Page$getPageImage$6.invokeSuspend(Page.kt:139)
	at suwayomi.tachidesk.manga.impl.Page$getPageImage$6.invoke(Page.kt)
	at suwayomi.tachidesk.manga.impl.Page$getPageImage$6.invoke(Page.kt)
	at suwayomi.tachidesk.manga.impl.util.storage.ImageResponse.getImageResponse(ImageResponse.kt:69)
	at suwayomi.tachidesk.manga.impl.Page.getPageImage(Page.kt:138)
	at suwayomi.tachidesk.manga.impl.Page.getPageImage$default(Page.kt:58)
	at suwayomi.tachidesk.manga.impl.Page.getPageImageDownload(Page.kt:189)
	at suwayomi.tachidesk.manga.impl.download.fileProvider.ChaptersFilesProvider.downloadImpl(ChaptersFilesProvider.kt:150)
	at suwayomi.tachidesk.manga.impl.download.fileProvider.ChaptersFilesProvider.access$downloadImpl(ChaptersFilesProvider.kt:67)
	at suwayomi.tachidesk.manga.impl.download.fileProvider.ChaptersFilesProvider$download$1.execute(ChaptersFilesProvider.kt:212)
	at suwayomi.tachidesk.manga.impl.download.fileProvider.ChaptersFilesProvider$download$1.execute(ChaptersFilesProvider.kt:212)
	at suwayomi.tachidesk.manga.impl.ChapterDownloadHelper.download(ChapterDownloadHelper.kt:49)
	at suwayomi.tachidesk.manga.impl.download.Downloader.run(Downloader.kt:155)
	at suwayomi.tachidesk.manga.impl.download.Downloader.access$run(Downloader.kt:40)
	at suwayomi.tachidesk.manga.impl.download.Downloader$run$1.invokeSuspend(Downloader.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:98)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:807)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
Caused by: java.io.IOException: canceled due to java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:593)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoSuchMethodError: 'android.text.Spanned android.text.Html.fromHtml(java.lang.String, int)'
	at keiyoushi.a.b.a.a(Unknown Source)
	at keiyoushi.a.b.a.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.extension.all.webtoons.z.intercept(Unknown Source)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.CloudflareInterceptor.intercept(CloudflareInterceptor.kt:48)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(UserAgentInterceptor.kt:21)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(UncaughtExceptionInterceptor.kt:18)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:332)
	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:232)
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:580)
	... 3 common frames omitted
20:44:11.814 [DefaultDispatcher-worker-7] DEBUG suwayomi.tachidesk.manga.impl.download.Downloader source(2522335540328470744) - downloadChapter(4153 - 39777 | state= Queued, tries= 0, progress= 0.0)) -- start
20:44:11.815 [DefaultDispatcher-worker-7] DEBUG suwayomi.tachidesk.manga.impl.chapter.ChapterForDownload(mangaId= 4153, chapterId= 39777, chapterIndex= 15)::asDownloadReady -- isMarkedAsDownloaded= false, dbPageCount= 115, downloadPageCount= 0
20:44:11.816 [DefaultDispatcher-worker-7] DEBUG suwayomi.tachidesk.manga.impl.chapter.ChapterForDownload(mangaId= 4153, chapterId= 39777, chapterIndex= 15)::asDownloadReady -- reset download status and fetch page list
```

</details>

### Expected behavior

Downloaded chapter.

### Actual behavior

Failure to download the chapter.

### Suwayomi-Server version

v2.3.2298

### Used database

H2

### Server operating system

ghcr.io/suwayomi/suwayomi-server:preview

### Server Desktop Environment

_No response_

### Server JVM version

ghcr.io/suwayomi/suwayomi-server:preview

### Used client name

WebUI

### Client version

r3379

### Used web browser

FF 140.13.0esr (64-bit)

### Client operating system

W11 E

### Other details

_No response_

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [x] I have checked the ongoing preview changelog of **[Suwayomi-WebUI](https://github.com/Suwayomi/Suwayomi-WebUI/blob/master/CHANGELOG.md)** and **[Suwayomi-Server](https://github.com/Suwayomi/Suwayomi-Server/blob/master/CHANGELOG.md)** and this bug has **NOT** been listed as fixed
- [x] I have written a short but informative title (ideally less than ~100 characters).
- [x] I have tried the troubleshooting guide described in [README.md](https://github.com/Suwayomi/Suwayomi-Server?tab=readme-ov-file#troubleshooting-and-support)
- [x] I have updated the (**[Suwayomi-WebUI](https://github.com/suwayomi/suwayomi-webui/releases/latest)** and **[Suwayomi-Server](https://github.com/suwayomi/suwayomi-server/releases/latest)**) to the latest versions
- [x] I have filled out all of the requested information in this form, including specific version numbers.
- [x] I understand that **Suwayomi does not have or fix any extensions**, and I **will not receive help** for any issues related to sources or extensions.

--- a/AndroidCompat/src/main/java/android/text/Html.java
+++ b/AndroidCompat/src/main/java/android/text/Html.java
@@ -17,7 +17,13 @@ import org.xml.sax.XMLReader;
 
 public class Html {
 
+    public static final int FROM_HTML_MODE_LEGACY = 0x00000000;
+
     public static Spanned fromHtml(String source) {
+        return fromHtml(source, FROM_HTML_MODE_LEGACY);
+    }
+
+    public static Spanned fromHtml(String source, int flags) {
         return new FakeSpanned(Jsoup.clean(source, Safelist.none()));
     }
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -44,6 +44,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - (**WebView**) Fix `kcef` causing the server startup to fail
 - (**Category**) Fix library/category counts not matching the actual number of manga due to duplicate category-manga rows
 - (**Migration/Extension**) Fix `extensionRepos` setting not being migrated to `extensionStores`
+- (**Extension**) Fixes author's notes not working in sources that offer them
 
 ## [v2.3.2243] - 2026-07-13
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -44,7 +44,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - (**WebView**) Fix `kcef` causing the server startup to fail
 - (**Category**) Fix library/category counts not matching the actual number of manga due to duplicate category-manga rows
 - (**Migration/Extension**) Fix `extensionRepos` setting not being migrated to `extensionStores`
-- (**Extension**) Fixes author's notes not working in sources that offer them
+- (**Extension/Android**) Implement fromHtml with flags for author's note support
 
 ## [v2.3.2243] - 2026-07-13
 

